recipes/pyrr/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pyrr" %}
+{% set version = "0.10.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "3c0f7b20326e71f706a610d58f2190fff73af01eef60c19cb188b186f0ec7e1d"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

No need for the arguments anymore
```suggestion
  script: "{{ PYTHON }} -m pip install .  -vv "
```